examples/aws-noaa-oisst-avhrr-only-example.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<p>Should we pin pystac_client to newer than a certain value? <code>&gt;0.6.0</code>?make sure to quote to escape shell interpretation.</p><p>Change <code>!</code> to <code>%</code> per jupyter/SMSL recommendations</p><p> </p><p>I'm looking to why I got a pip error</p><pre class="ql-syntax" spellcheck="false">ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
boto3 1.26.110 requires botocore&lt;1.30.0,&gt;=1.29.110, but you have botocore 1.29.76 which is incompatible.
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/asdi-examples/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Works fine in a fresh conda env. TODO for later is general addition of using Conda envs.